### DISCUSSION TITLE
Install Guide

### DISCUSSION THREAD

**Discussion by Arne91** [latex3/latex2e#1205] [Feedback on new functionality] [answered]
I want to start with latex on my linux computer. I used it in the past under windows. On the latex page https://www.latex-project.org/get/ is recommanded to install it via git. But there is no guide for installation or using. How can I compile a latex file with linux?

**Comment by u-fischer** [comment, marked as answer]
The page recommends to use a TeX distribution. git is only for developers who want to get the newest code. 

> If you’re new to TeX and LaTeX or just want an easy installation, get a full TeX distribution. 

https://tug.org/texlive/

**Reply by Arne91** [reply]
Ah ok. Thanks

**Comment by FrankMittelbach** [comment]
I'll make that information a bit clearer on that page by adding a prominent warning that this repository is not meant for end users who just want to process their documents.

**Reply by Arne91** [reply]
Ok thanks :) That sounds good.

**Reply by FrankMittelbach** [reply]
Went online a few minutes ago
